Source/WebGPU/WebGPU/Texture.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#import "WebGPU.h"
```

Comment:
This shouldn't be necessary. All .cpp files must `#include "config.h"` as their first include, and that file has `#include "WebGPU.h"` inside it.

Reply:
Yeah that I know, my editor is angry about the missing definitions though (which yeah, it will be there when the header file gets included into .cpp files)